src/AppBundle/Controller/FeedbackController.php

```diff
@@ -23,22 +23,61 @@ public function indexAction(Request $request)
         }
 
         if ($form->isSubmitted() && $form->isValid()) {
-            //Stores the submitted feedback
-            $em = $this->getDoctrine()->getManager();
             $feedback = $form->getData();
-            $feedback->setUser($user);
-            $em->persist($feedback);
-            $em->flush();
+            $this->sumbitFeedback($feedback);
+            $this->addFlash("success", "Tilbakemeldingen har blitt registrert, tusen takk!");
 
-            //Notifies on slack (NotificationCHannel)
-            $messenger = $this->container->get('AppBundle\Service\SlackMessenger');
-            $messenger->notify($feedback->getSlackMessageBody());
+            return $this->redirect($returnUri); //Makes sure the user cannot submit the same form twice (e.g. by reloading page)
+        }
+        //Stores the form data temporary in the session
+        $request->getSession()->set('feedbackFormData', $form->getData());
 
+        return $this->redirect($returnUri);
+    }
+    public function errorSubmitAction(Request $request)
+    {
+        $feedback = new Feedback;
+        $form = $this->createForm(ErrorFeedBackType::class, $feedback);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $feedback = $form->getData();
+            $this->sumbitFeedback($feedback);
             $this->addFlash("success", "Tilbakemeldingen har blitt registrert, tusen takk!");
-            
-            return $this->redirect($returnUri); //Makes sure the user cannot submit the same form twice (e.g. by reloading page)// Will also r
+        } else {
+            //Stores the form data temporary in the session
+            $request->getSession()->set('errorFeedbackFormData', $form->getData());
+        }
+
+        //Redirects to a copy of 500-errorpage, since we can't redirect back, as it will cause exception.
+        return $this->redirect($this->generateUrl('feedback_admin_error'));
+    }
+    //Stores feedback
+    private function sumbitFeedback(Feedback $feedback)
+    {
+        $user = $this->getUser();
+        //Stores the submitted feedback
+        $em = $this->getDoctrine()->getManager();
+        //Resets EntityManager if closed by exception
+        if (!$em->isOpen()) {
+            $this->getDoctrine()->resetManager();
         }
+        $feedback->setUser($user);
+        $em->persist($feedback);
+        $em->flush();
 
+        //Notifies on slack (NotificationCHannel)
+        $messenger = $this->container->get('AppBundle\Service\SlackMessenger');
```

Comment:
Forslag:
```suggestion
        $messenger = $this->container->get(SlackMessenger::class);
```

Krever at SlackMessenger er importert (hvilket den allerede er i denne filen)